### DISCUSSION TITLE
Add tests boilerplate for cypress/ react-testing library

### DISCUSSION THREAD

**Discussion by YannHulot** [Etheonor/supanextail#6] [Feedback]
Consider adding a few example tests for specs/tests with Cypress and/or react-testing-library.

Testing is hard, even today. Writing a few examples tests and maybe adding some documentation in the Readme regarding testing best practices, libraries suggestions, blog posts can help somebody new to testing get started. 

I think a non tested web app leads to issues down the road. The minimum would be to have a few smoke tests on each page to check that the page is loading, not broken ....

**Comment by Etheonor** [comment]
Just added Cypress on a branch and it's working great. I just added one test that visit the homepage and the pricing page but it's fine. The flow is setup for Github and Vercel deployment. I will add more tests this week
